docker-compose.abuse.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  abuse:
+    build:
+      context: ./docker/abuse
+      dockerfile: Dockerfile
+    container_name: abuse
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - ABUSE_LOG_LEVEL=${ABUSE_LOG_LEVEL}
+      - ABUSE_MAILBOX=${ABUSE_MAILBOX}
+      - ABUSE_SPONSOR=${ABUSE_SPONSOR}
+      - BLOCKER_API_URL=${BLOCKER_API_URL}
+      - BLOCKER_AUTH_HEADER=${BLOCKER_AUTH_HEADER}
+      - EMAIL_SERVER=${EMAIL_SERVER}
+      - EMAIL_USERNAME=${EMAIL_USERNAME}
+      - EMAIL_PASSWORD=${EMAIL_PASSWORD}
+      - MONGO_CONNECTIONSTRING=${MONGO_CONNECTIONSTRING}
```

Comment:
For this we also have env vars already - all of the `SKYNET_DB_*` ones.

While the `ACCOUNTS_EMAIL_URI` is only being used by `accounts` and we can change it if you really want, the DB ones are being used by a bunch of services by now (accounts, skyd, blocker, malware-scanner) and I don't think we'll change it.

Reply:
updated, but we should in my opinion migrate to use `MONGO_CONNECTIONSTRING` for two reasons:
1. it's more descriptive than `SKYNET_DB_`
2. with mongo it's just easier to use the connection string because you can plug that into a mongo client directly

Reply:
From offline:
We have different services which use different `readPreference`s, so this might be problematic.
We will discuss and figure out how to move forward.